schema/elasticsearch/embed.go

```diff
@@ -20,7 +20,7 @@
 
 package elasticsearch
 
-import _ "embed"
+import _ "embed" // enable embedding
```

Comment:
interesting decision to _add_ a comment... but tbh I agree with this, I think the blank embed-import is a very weird requirement.

Reply:
^ introduced by gofancyimports, and gci does not do this.  seems fine to keep / worth keeping though.